app/letters/utils.py

```diff
@@ -74,6 +85,33 @@ def upload_letter_pdf(notification, pdf_data, is_test_letter=False):
 
     current_app.logger.info("Uploaded letters PDF {} to {} for notification id {}".format(
         upload_file_name, bucket_name, notification.id))
+    return upload_file_name
+
+
+def move_scanned_pdf_to_letters_pdf_bucket(filename):
+    source_bucket_name = current_app.config['LETTERS_SCAN_BUCKET_NAME']
+    target_bucket_name = current_app.config['LETTERS_PDF_BUCKET_NAME']
+
+    s3 = boto3.resource('s3')
+    copy_source = {'Bucket': source_bucket_name, 'Key': filename}
+    target_filename = get_folder_name(datetime.utcnow()) + filename
+    target_bucket = s3.Bucket(target_bucket_name)
+    obj = target_bucket.Object(target_filename)
+    obj.copy(copy_source, ExtraArgs={'ServerSideEncryption': 'AES256'})
```

Comment:
does this move across tags? if so, does the expiry time get refreshed or stay the same?

Reply:
Tags get moved across 
The expiry time will get reset when moved to the new bucket, so a pdf that has been in the scan bucket for 3 days will be moved to the process bucket and will have the expiration time reset to 7 days.